perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -579,7 +579,7 @@ public void sendMessage(PerunSession sess, Application app, MailType mailType, S
 		if (mail == null) throw new RegistrarException("Notification template for "+mailType+" is not defined.");
 		if (!mail.getSend()) throw new RegistrarException("Sending of notification "+mailType+" is disabled.");
 
-		if (!AuthzResolver.hasRole(sess.getPerunPrincipal(), Role.PERUNADMIN)) {
+		if (!(AuthzResolver.hasRole(sess.getPerunPrincipal(), Role.PERUNADMIN) || AuthzResolver.hasRole(sess.getPerunPrincipal(), Role.PERUNADMINBA))) {
```

Comment:
This call effectively does the same as isPerunAdmin()

Reply:
Well, I think it is different because it does throw the error if the role does not exist in the Perun. `isPerunAdmin()` does not check if the role exists.